arduino/health_monitor/health_monitor_core.cpp

```diff
@@ -77,7 +77,7 @@ HealthState treament_update(HealthState health_state) {
 PlayerState new_player_state() {
     PlayerState player;
     player.tick                  = 0;
-    player.health.health         = 2;
+    player.health.health         = to_health(StateBounds::SUPER_HEALTHY);
```

Comment:
Did you check to see if this persists or resets?

Reply:
I did not, but the initialization is happening in `setup`, not as part of some global var init... this will be true for the rev2 setup as well, since init has to happen in the main `setup` function.